service/debugger/debugger.go

```diff
@@ -906,6 +907,25 @@ func (d *Debugger) SetVariableInScope(scope api.EvalScope, symbol, value string)
 	return s.SetVariable(symbol, value)
 }
 
+func (d *Debugger) getRunningGoroutines(count int) map[int]*proc.G {
+	threads := d.target.ThreadList()
+	sort.Slice(threads, func(i, j int) bool { return threads[i].ThreadID() < threads[j].ThreadID() })
```

Comment:
What does sorting buy us here?

Reply:
The output of ThreadList is not guaranteed to always be in the same order (because the native backend is converting from a map).

Reply:
Right, but we just iterate them and them stuff them into another map and return it, I don't see why we should sort them first, order doesn't really matter here unless I'm missing something obvious.

Reply:
Is this to make the call idempotent when called with a same `count` value?

Reply:
Yes.